src/angularjs/src/app/index.route.js

```diff
@@ -20,7 +20,7 @@
                 template: '<ui-view/>'
             })
             .state('places.list', {
-                url: '',
+                url: ':place1/:place2/:place3/',
```

Comment:
Not sure what you mean by adding from the compare page; like another search drop-down? From the prototype, it seemed to me the intent was to go back and forth between the list and compare pages with some set of selections.

Another option would be to handle the selections internally with a service, but then the selections would be lost on page refresh and direct linking wouldn't work (not sure if we care about either of those cases). If they haven't selected anything yet, the URL just has a couple of extra slashes on the end.